### ISSUE TITLE
Error: cannot overwrite multiple values with a single value when using a self-hosted runner

### ISSUE THREAD

**Issue by vmalloc** [de-vri-es/setup-git-credentials#9]
```
Run fusion-engineering/setup-git-credentials@v2
  with:
    credentials: ***
git config --global credential.helper store
git config --global url.https://github.com/.insteadOf ssh://git@github.com/
warning: url.https://github.com/.insteadof has multiple values
error: cannot overwrite multiple values with a single value
       Use a regexp, --add or --replace-all to change url.https://github.com/.insteadOf.
Error: The process 'git' failed with exit code 5
```

**Comment by de-vri-es** [comment]
Hey! Thank you for reporting this.

This was fixed in 2.0.3: https://github.com/fusion-engineering/setup-git-credentials/releases/tag/v2.0.3

I recommend updating to the latest version (2.0.6 currently). If that doesn't solve your issue, please let me know and we can investigate further!

**Comment by vmalloc** [comment]
thanks!
